fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
Codacy found an issue: ['postVariable' is not defined.](http://dev.k8s.dev.codacy.org/manual/devk8scodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=651)